_episodes/52.md

```diff
@@ -0,0 +1,127 @@
+
+* Faster reading of ORC files in Hive
+* More types supported for schema evolution in Hive
+* Security improvements, including logging out of a session with the Web UI
+
+[Trino 430](https://trino.io/docs/current/release/release-430.html)
+
+* Improved performance of `GROUP BY`
+* Support for setting a timezone on the session level
+* Table statistics in MariaDB
+
+## Concept of the episode: JDBC-based connectors
+
+In Trino, we have a lot of connectors that are based on top of JDBC. JDBC could
+stand for "just da best connectors," but it's really Java database connectivity,
+and it's one of the core APIs by which many of the most prominent connectors in
+the Trino ecosystem function. It's so common, in fact, that we have
+[an example JDBC connector in Trino](/docs/current/develop/example-jdbc.html) to
+make it easier to go implement your own JDBC-based connector if you need one.
+
+## Concept of the episode: YugabyteDB
+
+But if the topic of today's episode is YugabyteDB, why are we talking about
+PostgreSQL? Well, if you're unfamiliar with Yugabyte, lifting from
+[their docs](https://docs.yugabyte.com/): "YugabyteDB is distributed PostgreSQL
+that delivers on-demand scale, built-in resilience, and a multi-API interface."
+Distributed architecture should be a familiar concept to a community involved
+with a distributed query engine, and if you understand how Trino is able to
+leverage it, you should also understand why it makes sense to pair with
+Yugabyte. We'll be discussing why Yugabyte got started, what it does differently
+from other databases, what it does better than other databases, and how you
+might want to use it with Trino.
+
+## Demo of the episode: Trino on YugabyteDB
+
+As part of the episode, we'll also be showing off how you can use YugabyteDB 
+with Trino. We start with using the PostgreSQL connector, then Denis shows how 
+to use the Postgres connector to run Trino with Yugabyte. It's always hard to
+explain demos in show notes, so tune into the YouTube video and take a look for
+yourself if you're curious!
+
+## Rounding out
+
+Trino Summit, the biggest Trino event of the year, is coming up on the 13th and
+14th of December, and like Trino Fest, it'll be fully virtual. If you'd like to
+give a talk about anything related to Trino, we're looking for speakers now.
+[Submit your talk here!](https://sessionize.com/trino-summit-2023/) If you'd
+rather attend, you can also
+[go register to attend now](https://www.starburst.io/info/trinosummit2023/?utm_source=trino&utm_medium=website&utm_campaign=NORAM-FY24-Q4-EV-Trino-Summit-2023&utm_content=tcb).
+
+Prior to Trino Summit, if you'd like to learn about SQL from the absolute
+experts, we've also gotten started with the [Trino Training Series](/blog/2023/09/27/training-series)
+that we'll be running as a buildup to the summit. Register now and look forward
+to four great sessions starting from the ground up and ending with some key
```

Comment:
Link to the blog post about the first one and mention three more .. also should we update the announcement blog post to link to the blog posts about each one as a separate PR?